Source/ULibs.TinyJsonDeser/RELEASENOTES.md

```diff
@@ -22,4 +22,4 @@
 
 ### Features
 
-- A lightweight thread-safe JsonDeserializer class that can deserialise json text to .NET types. It does not support binding to specific .NET types. Rather, objects are deserialised to instances of `IDictionary<string, object>`, arrays are deserialised to instances of `object[]`, and literals are deserialised to either a `bool`, `double`, `string` or `null`.
+- A lightweight thread-safe JsonDeserializer class that can deserialize json text to .NET types. It does not support binding to specific .NET types. Rather, objects are deserialized to instances of `IDictionary<string, object>`, arrays are deserialized to instances of `object[]`, and literals are deserialized to either a `bool`, `double`, `string` or `null`.

```

Comment:
You might want to add a version bump with a Fix entry describing the change. It's trivial, but this is public, so... :)

Reply:
Same goes for the other release notes in this PR. If you don't bump the version, the change won't be released.